www/scripts/main.js

```diff
@@ -229,6 +230,11 @@ function setLastRoomName(roomName) {
 
 /* Helpers */
 
+function displayMessage(recipient) {
```

Comment:
No I totally agree and noticed it post-merge. I was in .Net mode where it would be obvious that it returns a bool and so the name would be better (although still probably not great). shouldDisplayMessage is good.